params/jack_bird.R

```diff
@@ -0,0 +1,12 @@
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4,67
```

Comment:
Please use a dot instead of comma
```suggestion
sig2 <- 4.67
```